README.md

```diff
@@ -106,24 +110,49 @@ load). The sample service-worker-registration.js file
 the service worker lifecycle event you can listen for to trigger this message.
 
 
-## Methods
+### Command-line interface
+
+For those who would prefer not to use `sw-precache` as part of a `gulp` or `Grunt` build, there's a
+[command-line interface](cli.js) which supports all the same options, provided via flags. Sensible
+defaults are assumed for options that are not provided.
+
+For example, if you are inside the top-level directory that contains your site's contents, and you'd
+like to generate a `service-worker.js` file that will automatically precache all of the local
+files, you can simply run
+
+```sh
+$ sw-precache
+```
+
+Alternatively, if you'd like to only precache `.html` files that live within `dist/`, which is a
+subdirectory of the current directory, you could run
+
+```sh
+$ sw-precache --root=dist --static-file-globs='dist/**/*.html'
+```
+ 
+(Be sure to use quotes around parameter values, like `*`, that have special meanings to your shell.)
+
+## API
+
+### Methods
 
 The `sw-precache` module exposes two methods: `generate` and `write`.
 
-### generate(options, callback)
+#### generate(options, callback)
 `generate` takes in [options](#options), generates resulting service worker code as a string, and
 then invokes `callback(error, serviceWorkerString)`.
 In the 1.x releases of `sw-precache`, this was the default and only method exposed by the module.
 
 Since 2.2.0, `generate()` also returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
 
-### write(filePath, options, callback)
+#### write(filePath, options, callback)
 `write` is a helper method that calls `generate` and takes the resulting string content and
 writes it to disk, at `filePath`. It then invokes `callback(error)`.
 
 Since 2.2.0, `write()` also returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
 
-## Options
+### Options Parameter
```

Comment:
If "Options Parameter" is prefixed by `###` now, then shouldn't all the sub-items that follow it be prefixed with `####`? E.g. `### cacheId` should change to `#### cacheId`, etc.?